modules/ROOT/pages/browser-support.adoc

```diff
@@ -0,0 +1,26 @@
+= Browser Support for Anypoint Platform
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+// page-aliases: 
+
+Use Anypoint Platform with the latest version of any supported browser:
+
+* Microsoft Edge with Chromium
+* Microsoft Edge wihtout Chromium (supported through December 31, 2020)
+* Google Chrome
+* Mozilla Firefox
+* Safari
+
+The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```

Comment:
```suggestion
Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```
Will your audience be familiar with SFLE policy and what this means?

Reply:
A significant portion will not. Link is in the See Also.